docs/guides/importing-private-repositories.rst

```diff
@@ -0,0 +1,125 @@
+Manually Importing Private Repositories
+=======================================
+
+.. warning::
+
+   This guide is for users of :doc:`/commercial/index`.
+   **If you are using GitHub, GitLab, or Bitbucket**,
+   we recommend :doc:`connecting your account </connected-accounts>` and importing your project from
+   https://readthedocs.com/dashboard/import instead of importing it manually.
+
+If you are using an unsupported integration, or don't want to connect your account,
+you'll need to do some **extra steps in order to have your project working**.
+
+#. **Manually import your project using an SSH URL**
+#. **Allow access to your project using an SSH key**
+#. **Setup a webhook to build your documentation on every commit**
+
+.. contents:: Table of contents
+   :local:
+   :backlinks: none
+   :depth: 3
+
+Importing your project
+----------------------
+
+#. Go to https://readthedocs.com/dashboard/import/manual/
+#. Fill the :guilabel:`Repository URL` field with the SSH form of your repository's URL,
+   e.g ``git@github.com:readthedocs/readthedocs.org.git``
+#. Fill the other required fields
+#. Click :guilabel:`Next`
+
+Giving access to your project with an SSH key
+---------------------------------------------
+
+After importing your project the build will fail,
+because Read the Docs doesn't have access to clone your repository.
+To give access, you'll need to add your project's public SSH key to your VCS provider.
```

Comment:
Regarding UX, it would be good to do something different here to avoid the initial build to fail:

1. do not trigger the first build if the project is imported manually
2. on manual imports, add an extra step that shows the just generated project's public SSH key to the user for them to copy and paste into their VCS service

I'd like if we can implement 2) on the new templates to make the onboarding process nicer. cc @agjohnson